host/tests/L3_TestCases/dsFPD/dsFPDHelperClass.py

```diff
@@ -53,16 +53,17 @@ def __init__(self, testName:str, qcId:str, log:logModule=None ):
 
```

Comment:
defaults has to be removed from testSetup file.. check dsAudio

Reply:
Removed the defaults and artifacts from test Setup file.

Reply:
added back as the test was failing without proper keys.
Removed unwanted lines from it.